apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UpdateObjectOptions.cs

```diff
@@ -88,6 +88,12 @@ public sealed class UpdateObjectOptions
         /// </summary>
         public RetryOptions RetryOptions { get; set; }
 
+        /// <summary>
+        /// Must be true to remove the retention configuration, reduce its unlocked retention period, or change its
+        /// mode from unlocked to locked.
+        /// </summary>
+        public bool? OverrideUnlockedRetention { get; set; }
```

Comment:
You are not unit testing this change specifically.

Reply:
+1 for this.